src/backend/commands/createas.c

```diff
@@ -471,16 +470,9 @@ ExecCreateTableAs(ParseState *pstate, CreateTableAsStmt *stmt,
 		/* get object address that intorel_startup saved for us */
 		address = ((DR_intorel *) dest)->reladdr;
 
-		/* save the rowcount if we're given a completionTag to fill */
-		if (completionTag)
-			snprintf(completionTag, COMPLETION_TAG_BUFSIZE,
-					 "SELECT " UINT64_FORMAT,
-					 queryDesc->es_processed);
-=======
 		/* save the rowcount if we're given a qc to fill */
 		if (qc)
-			SetQueryCompletion(qc, CMDTAG_SELECT, queryDesc->estate->es_processed);
->>>>>>> 4dbcb3f844eca4a401ce06aa2781bd9a9be433e9
+			SetQueryCompletion(qc, CMDTAG_SELECT, queryDesc->estate->es_processed)
```

Comment:
```suggestion
			SetQueryCompletion(qc, CMDTAG_SELECT, queryDesc->es_processed);
```
I've already cherry-picked 2f9661311b83dc481fc19f6e3bda015392010a40 for another task some time ago, and it appeared that `queryDesc->estate` was null, so we need to return to usage of `queryDesc->es_processed`.

Reply:
> it appeared that `queryDesc->estate` was null

I don't think so: compare the `ExecCreateTableAs` function with the vanilla 13 version: the dereference of `queryDesc->estate->es_processed` happens BEFORE `ExecutorFinish`!

Reply:
Hm, there is a comment right above, seems related:
```
		/*
		 * In GPDB, computing the row count needs to happen after ExecutorEnd()
		 * because that is where it gets the row count from dispatch. There's
		 * also some special processing if the relation was a replicated table.
		 * In upstream Postgres, the rowcount is saved before ExecutorFinish().
		 */
```
So I suggest to double check this moment. And, anyway, `;` is missing.

Reply:
So we need to keep this comment, not delete it.

Reply:
it was not deleted.

Reply:
`queryDesc->estate->es_processed` is used at all commits, before sync and after.

Reply:
Ok. But please double check `CTAS` command is Ok once compilation is fixed.